refactor default value in verticalGradientStops.kt refactor colors in PrimaryInputField.kt

maryam2070 opened the pull request.

maryam2070 committed:
refactor default value in verticalGradientStops.kt
refactor colors in PrimaryInputField.kt

moashrafff commented on chatia-ui-components/src/commonMain/kotlin/com/chatia/ui/components/inputFields/PrimaryInputField.kt:
```diff
@@ -39,11 +42,13 @@ import com.chatia.ui.components.texts.PrimaryText
     modifier: Modifier = Modifier,
-    value: String,
-    onValueChange: (String) -> Unit,
+    value: String="llll",
```
what does "llll" mean?

moashrafff commented on chatia-ui-components/src/commonMain/kotlin/com/chatia/ui/components/inputFields/PrimaryInputField.kt:
```diff
@@ -39,11 +42,13 @@ import com.chatia.ui.components.texts.PrimaryText
-    value: String,
-    onValueChange: (String) -> Unit,
+    value: String="llll",
+    onValueChange: (String) -> Unit={},
```
why default values for value and on valuechange?

moashrafff commented on chatia-ui-components/src/commonMain/kotlin/com/chatia/ui/components/verticalGradientStops.kt:
```diff
@@ -26,7 +26,11 @@ import androidx.compose.ui.graphics.Color
 fun Modifier.verticalGradientStops(
-    vararg stops: Pair<Float, Color>
+    vararg stops: Pair<Float, Color> = arrayOf(
+        0.12f to Color(0xFFFFFFFF),
```
use material color theme and not static colors

maryam2070 committed:
remove default values from PrimaryInputField.kt
add static colors to ExtendedColors.kt and use them in verticalGradientStops.kt
